articles/Choosing-between-Django-Flask-and-FastAPI/index.md

```diff
@@ -0,0 +1,102 @@
+- Flask has a singular source that implies that it will handle every request in turns, one after the other, so regardless of how many multiple requests it still takes them in turns, it takes more time.
+
+##### Flask: Use cases
+Flask can be used for commercial projects. It can help you get started quickly but doesn't work well for resembling a real load. Projects you can easily implement in flask are : E-commerce systems, Facebook/ Twitter bot, An online social network, static sites, etc. but it's not advisable to be use it for high-load enterprise software.
+
+#### FastAPI  
+Fast API is a modern open-source fast and highly performant Python web framework for building Web APIs and is based on Python 3.6+ standard type hints. 
+
+##### Pros
+- Fast API validates the developer's data type even in deeply nested JSON requests.
+- Fast API is built on standards like JSON Schema(a tool for validating the structure of JSON data), OAuth 2.0(it's the industry-standard protocol for authorization), and OpenAPI(which is a publicly available application programming interface)
+- FastAPI makes it easy to build a GraphQL API with a Python library called *graphene-python*.
+
+#### Cons
+- Because FastAPI is relatively new, the community is small compared to other frameworks and regardless of its detailed documentation there are few external educational materials.
+
+##### FastAPI Use cases
+Fast API which is relatively new amongst the three, works perfectly if your concern is speed. Netflix uses it for its internal crisis management. It also scales perfectly in deploying production-ready machine learning models because ML models work best in production when they are wrapped around a REST API and deployed in a microservice.
+
+Now we're going to compare Django, Flask, and FastAPI based on their *packages*, *community*, *performance*, *flexibility*, *job opening*, and *education*.
+
+**Packages**
+Among Django, Flask, and FastAPI, Django has the most packages that enable reusability of code. It is a full-stack web development framework unlike Flask and FastAPI which are minimalistic frameworks for building fast websites.
+
+**Community**
+Django has the biggest community because of its wide use and popularity next to the Flask which also has a thriving community. FastAPI on the other hand has a small community because it's fairly new.
+
+**Performance**
+In performance, FastAPI leads because it is speed-oriented then next to Flask and finally Django which is not very fast.
+
+**Flexibility**
+Flexibility is something developers value a lot, and Flask is more flexible than Django. Fast API, on the other hand is flexible code-wise and doesn't restrict the code layout. So we can say Flask is the most flexible among all of the above.
+
+**Job openings**
+No doubt they are more job openings in the python web ecosystem with Django requirement next to Flask and finally Fast API which is few, so if your is aim to get a job quickly then Django should be your go-to guy.
```

Comment:
no doubt there are. I would suggest checking for such minor errors throughout the article to receive the complete payout. I believe another round of review on this article will result in a tiered payout.